ledger-api/grpc-definitions/com/daml/ledger/api/v1/connected_domains_service.proto

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+option java_outer_classname = "ConnectedDomainsServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Allows clients to get state of connected domains from the ledger.
+service ConnectedDomainsService {
+  // Get the list of connected domains.
+  rpc GetConnectedDomains (GetConnectedDomainsRequest) returns (GetConnectedDomainsResponse);
+}
+
+message GetConnectedDomainsRequest {
+  // The submitting party
+  // Must be a valid PartyIdString (as described in ``value.proto``).
+  // Required
+  string party = 1;
+
+  ConnectedDomainsFilter connected_domain_filter = 2;
```

Comment:
s/connected_domain_filter/connected_domains_filter ?
(we should change on our side as well)